src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -587,7 +587,7 @@ def test_aks_create_and_update_with_vpa(
     @AKSCustomResourceGroupPreparer(
         random_name_length=17, name_prefix="clitest", location="westus2"
     )
-    def test_aks_create_with_addon_autoscaling(
+    def test_aks_create_with_optimized_addon_scaling(
```

Comment:
Please update the name of corresponding recording file

Reply:
update!